### PR TITLE
Rewrite "array aggregate" sentence, replace `()` with `[]`

### PR DESCRIPTION
**Pull request by reznikmm**

### CODE REVIEW

File: content/courses/ada_2022_whats_new/chapters/array_aggregates.rst [hunk 1]
```diff
@@ -37,31 +37,54 @@ single-element aggregates. Consider this:
...
+and values, just like named component associations:
+
+.. code-block::
+
+   Data : Integer_Array (1 .. 5) :=
+     [for J in 2 .. 3 => J, 5 => 5, others => 0];
+
+Here :ada:`Data` contains :ada:`(0, 2, 3, 0, 5)`, not :ada:`(2, 3, 5, 0, 0)`.
+
+Another form of iterated component association correspond to a positional
```
Comment (yannickmoy) [inline content/courses/ada_2022_whats_new/chapters/array_aggregates.rst:80]: "corresponds"

**Review by yannickmoy — CHANGES_REQUESTED** [review]
see typo

File: content/courses/ada_2022_whats_new/chapters/array_aggregates.rst (hunk 1 above)
Comment (reznikmm) [reply content/courses/ada_2022_whats_new/chapters/array_aggregates.rst]: Fixed and rebased. Thank you

**Review by yannickmoy — APPROVED** [review]

**Comment by gusthoff** [comment]
Thanks for the review, @yannickmoy!

**Pull request merged**